src/client/api/pythonApi.ts

```diff
@@ -164,6 +168,7 @@ export class PythonExtensionChecker implements IPythonExtensionChecker {
         if (this.waitingOnInstallPrompt) {
             return this.waitingOnInstallPrompt;
         }
+        PythonExtensionChecker.promptDispalyed = true;
```

Comment:
THis is the easiest way.
The problem is Jupyter extesnion actiavtes before the tests run, and as part of jupyter activation we might have pre-warming variables and the like which could cause python ext prompt to be dispalyed.
Hence there no way to stub or do anything, except capture a flag to indicate whether the prompt was displayed.